salt/metalk8s/salt/master/configured.sls

```diff
@@ -12,6 +13,7 @@ Configure salt master:
     - backup: false
     - template: jinja
     - defaults:
+        debug: {{ metalk8s.debug | tojson }}
```

Comment:
is the `| tojson` filter really necessary ?

Reply:
I think I had issues with this value being interpreted as a string, but I'll see if I can remove it :)

Reply:
Removed!